src/lib/core/sio.c

```diff
@@ -43,6 +43,10 @@
 #include "uri/uri.h"
 #include "errinj.h"
 
+#if TARGET_OS_DARWIN
```

Comment:
Why does the test requires root privileges? ERRINJ, which replaces the value of `somaxconn` variable?

Reply:
Because increase of the systemwide limit only works under root. Having the errinj of such kind would only test the errinj itself, which isn't very handy. Testing the actual system limit really isn't possible automatically (in our test env).

Reply:
Resolving since the PR is approved.